src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -598,7 +581,7 @@ public TypeSymbolWithAnnotations SetUnknownNullabilityForReferenceTypesIfNecessa
         public TypeSymbolWithAnnotations WithTopLevelNonNullability()
```

Comment:
>WithTopLevelNonNullability [](start = 41, length = 26)

Not related to this PR: We might rename to `WithTopLevelNonNullabilityForReferenceTypes`, which would be consistent with methods below and above.